config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: quay.io/luzuccar/node-observability-operator
+  newTag: v0.0.1
```

Comment:
@lmzuccarelli 
Do you think it's possible to implement what @josefkarasek suggested, making the tag and image get updated from $IMG and commitID?

Reply:
Yes I'll implement that

Reply:
Actually I'm not so sure anymore. 
I need to substitute this in the CI for the image built in CI ...